docs/notifications/index.md

```diff
@@ -158,6 +158,21 @@ You can customize the origin names shown in the UI by passing an object where th
 
 Each notification processor will receive its own row in the settings page, where the user can enable or disable notifications from that processor.
 
+### Automatic notification cleanup
+
+Notifications are deleted automatically after a certain period of time to prevent the database from growing indefinitely
+and to keep the user interface clean. The default retention period is set to 1 year, meaning that notifications older
+than that will be deleted automatically.
+
+The retention period can be configured by setting the `notifications.retention` in the `app-config.yaml` file.
+
+```yaml
+notifications:
+  retention: 1y
```

Comment:
Asking because we're doing some task related stuff as well in this format can you do:

`1y 6m1 0d`? or `3m`? or even `700d`?

Just wondering 😅

Reply:
Well, you can do it couple of ways the `readDurationFromConfig` supports:

- A string in the format of '1d', '2 seconds' etc. as supported by the `ms`
  library.
- A standard ISO formatted duration string, e.g. 'P2DT6H' or 'PT1M'.
- An object with individual units (in plural) as keys, e.g. `{ days: 2, hours: 6 }`.

Reply:
Apparently, https://github.com/vercel/ms does not support having multiple units in a string; in that case, you would need to create an object.